3 Reports.sql

```diff
@@ -0,0 +1,19 @@
+--1.
+select * from BikeShop where CustomerAddress like '%Spring Valley%' or CustomerAddress like '%Monsey%'
```

Comment:
Break up select statements onto separate lines (select, from, where...) Fix in all questions